src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -50,6 +50,12 @@ public class OverseasEntitiesController {
     @Value("${FEATURE_FLAG_ENABLE_VALIDATION_25082022}")
     private boolean isValidationEnabled;
 
+    @Value("${FEATURE_FLAG_ROE_UPDATE_24112022}")
```

Comment:
could you include the word ENABLE in the feature flag names, it should make it clear that the true/false is to enable/disable and keeps it aligned to our current one.
so 
FEATURE_FLAG_ROE_UPDATE_24112022 -> FEATURE_FLAG_ENABLE_ROE_UPDATE_24112022
FEATURE_FLAG_ROE_REMOVE_24112022 -> FEATURE_FLAG_ENABLE_ROE_REMOVE_24112022

will need to update the docker PR as well to match the names

Reply:
Sure, will update the PR's to add it in